manim/utils/bezier.py

```diff
@@ -21,14 +21,15 @@
 import typing
 from functools import reduce
 
+import bezier as bz
 import numpy as np
 from scipy import linalg
 
 from ..utils.simple_functions import choose
 from ..utils.space_ops import cross2d, find_intersection
 
 
-def bezier(
+def classic_bezier(
```

Comment:
The implementation of `get_quadratic_approximation_of_cubic` caused some headache: when I tried to replace the current implementation of `bezier`, a test using `get_quadratic_approximation_of_cubic` started to fail because the points  used in that function are not just 1d-numpy arrays as usual, but instead 2d ones. If someone wants to take a stab at fixing this, please go ahead -- then this `classic_bezier` function could also just be removed.